plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -337,89 +338,53 @@ export class PersonState {
             })
             return undefined
         }
-        return await this.mergeWithoutValidation(otherPersonDistinctId, mergeIntoDistinctId, teamId, timestamp, 0)
+        return promiseRetry(
```

Comment:
I think it's a good change but just wanted to clarify, previously we were retrying as fast as possible, but now we wait 1000ms between each? And that's the only functional change here?

Reply:
yes that was my intention